src/controllers/adminController.ts

```diff
@@ -186,7 +193,8 @@ export class AdminController implements AdminControllerInterface {
 
 		let authUsersResult: User[];
 		try {
-			authUsersResult = await getUsers(req.cookies['Authorization']);
+			const authToken = this._requestAuth.getAuthToken(req);
+			authUsersResult = await this._requestAuth.authApi.getUsers(authToken);
```

Comment:
This a real nit but is there any reason why you don't want to write the 2 lines above as one line? i.e:
```
authUsersResult = await this._requestAuth.authApi.getUsers(this._requestAuth.getAuthToken(req));
```

Reply:
I thought the line was getting a bit long with `.requestAuth.authApi.getUsers....` I don't mind having the function call as the param to getUsers though

I'll update in the next commit